openbb_terminal/economy/fred_model.py

```diff
@@ -348,3 +384,57 @@ def get_cpi(
     df = df.dropna()
 
     return df
+
+
+@log_start_end(log=logger)
+@check_api_key(["API_FRED_KEY"])
+def get_usd_liquidity(overlay: str = "SP500", show: bool = False) -> pd.DataFrame:
```

Comment:
default the overlay to none so that by default, this only gets the calculated series

Reply:
Done.

![Screenshot 2023-07-05 at 11 34 22 AM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/d57c68af-efe2-4826-9cbb-22203edfa36a)